### PR TITLE
Add improvements to the readme.

### PR DESCRIPTION
**Pull request by LeeBergstrand**
Add improvements to the readme.

### CODE REVIEW

**Comment by LeeBergstrand** [comment]
@jmtsuji Any other changes?

File: README.md [hunk 1]
```diff
@@ -60,15 +67,51 @@ The pipeline performs long read QC, assembly, end repair, polishing, contig rota
...
+
+rotary run_one -l s1_long.fastq.gz -r1 s1_R1.fastq.gz -r2 s1_R2.fastq.gz -d ../rotary_db_dir
+```
+**Note**: If you are using older nanopore flow cells you should stop the run, modify the config file 
+(see **Advanced Usage** below) and restart the run using the `rotary run` command.
+
+### 2b. Run Multiple Samples
+
+Rotary can target a directory containing numerous FASTQ files derived from various samples.
+It effortlessly organizes these files into sets corresponding to each sample and constructs a project
```
Comment (jmtsuji) [inline README.md:92]: Maybe OK to remove "effortlessly" for conciseness?

File: README.md [hunk 2]
```diff
@@ -151,31 +162,32 @@ please feel free to use this basic working version.
 
 ### _rotary_ workflow summary
 
-1. Perform simple long read QC using
+1. Perform simple short read QC (e.g., score, length, and adapter trimming) using [bbduk.sh from bbmap](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
```
Comment (jmtsuji) [inline README.md:165]: OK to remove "simple"? I think we are doing a fairly robust short read QC at this point.

File: README.md [hunk 3]
```diff
@@ -151,31 +162,32 @@ please feel free to use this basic working version.
...
+    Polypolish (if short reads were provided) or medaka (if only long reads were provided)
+11. Gene prediction via [DFAST](https://github.com/nigyta/dfast_core)
+12. Functional and taxonomic annotation via [EggNOG-mapper](https://github.com/eggnogdb/eggnog-mapper)
     and [GTDB-Tk](https://github.com/Ecogenomics/GTDBTk)
 
 ### Known issues
 
-- Short read QC is not performed (you have to do it in advance)
 - Limited flags are exposed for some key tools in the pipeline (e.g., Flye)
 - Minimum supported length for circular contigs is 100 kb (it should be a fairly easy fix in a future version to
```
Comment (jmtsuji) [inline README.md:187]: This issue can be deleted now (addressed this when porting repair.py to python)

File: README.md [hunk 4]
```diff
@@ -151,31 +162,32 @@ please feel free to use this basic working version.
...
   decrease this threshold)
 - Edge case: If you get "unlucky" and genome rotation is not substantial (e.g., _dnaA_ is already at the end of the
-  contig, re-polishing will have effect in improving assembly quality). Ideally, I should add a test of how the contig
+  contig, re-polishing will have an effect on improving assembly quality). Ideally, I should add a test of how the contig
```
Comment (jmtsuji) [inline README.md:190]: I made a grammar mistake here before: should be:

contig), re-polishing will have limited effect on improving assembly quality. Ideally, I should add a test of how the contig

File: README.md (hunk 3 above)
Comment (jmtsuji) [reply README.md:187]: Also, maybe can add another known issue? See below:

- If rotary is run in batch mode, then all samples in one batch must have only long reads, or all samples in one batch must have long and short reads. In future, we hope to make the pipeline more flexible in this area.

**Review by jmtsuji — CHANGES_REQUESTED** [review]
Thanks for the README update! I made a few minor suggestions, but otherwise looks good!

File: README.md (hunk 1 above)
Comment (LeeBergstrand) [reply README.md:92]: This is what happens when you use chatgpt to rewrite you stuff for proof editing purposes lol. It tends to use flowery sales language by default. I think the original was "automatically" and it changed it to "effortlessly", which is better for sales.

File: README.md (hunk 4 above)
Comment (LeeBergstrand) [reply README.md]: Addressed.

File: README.md (hunk 3 above)
Comment (LeeBergstrand) [reply README.md:187]: Thanks for the additions.

File: README.md (hunk 1 above)
Comment (LeeBergstrand) [reply README.md]: I changed it back to automatically.

**Comment by LeeBergstrand** [comment]
@jmtsuji I have addressed your feedback. I'm looking forward to your final approval.

**Review by jmtsuji — APPROVED** [review]
Looks good to me -- feel free to merge!

**Pull request merged**